releasenotes/notes/truncated_count_metric-ef102f7deabb9206.yaml

```diff
@@ -0,0 +1,25 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+upgrade:
+  - |
```

Comment:
Nit: Just a note for the future that I think you only need to include the keys you are actually setting (in this case `features`).